addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -580,17 +582,25 @@ def _move_model_in_data(env, ids_map, old_model, new_model):
     ]
     for old_id, new_id in ids_map:
         for rename in renames:
-            openupgrade.logged_query(
-                env.cr, """
+            query = """
                 UPDATE {table}
-                SET {field1} = '{new_value1}', {field2} = {new_value2}
-                WHERE {field1} = '{old_value1}' AND {field2} = {old_value2}
-                """.format(
-                    table=rename[0], field1=rename[1], field2=rename[2],
-                    old_value1=old_model, new_value1=new_model,
-                    old_value2=old_id, new_value2=new_id,
-                )
-            )
+                SET {field1} = %(new_value1)s, {field2} = %(new_value2)s
+                WHERE {field1} = %(old_value1)s AND {field2} = %(old_value2)s"""
+            if rename[0] == 'mail_followers':
```

Comment:
To avoid the loss of performance, maybe you could do it by not adding this specific part, and then later, in the end of `migration_voucher_moves(env)` script, do a cleanup of repeated followers?

Reply:
That can't be possible, as the unique constraint will rise then and break the upgrade process.